redux/auth/__snapshots__/saga.test.ts.snap

```diff
@@ -132,6 +132,64 @@ Object {
 }
 `;
 
+exports[`auth/saga loginWithFacebookSaga loginWithGoogleSaga handles successful signup through google 1`] = `
```

Comment:
Here you can see example of the nested `describe`. Should be fixed after change.